### ISSUE TITLE
Set fan speed step?

### ISSUE THREAD

**Issue by salmoaz** [mill1000/midea-ac-py#234]
I'd like to be able to set the fan step (5% increments, in my case), so that the slider I'm using on my dashboard will only show clean values.

**Comment by mill1000** [comment]
The card you use on your dashboard might be able to do this for you. 

Otherwise it's a pretty trivial feature to implement.

**Comment by mill1000** [comment]
So I experimented with adding a fan step option here: https://github.com/mill1000/midea-ac-py/tree/feature/fan_speed_step

But the behavior is probably not what you desire. Since the minimum fan speed is 1, all the steps are based off that value.

e.g. Step size = 10, the slider goes 1, 11, 21, 31, etc

**Comment by salmoaz** [comment]
Thanks for the quick response, I appreciate that you took the time to take a look at this. I did eventually realize that I was able to accomplish my end goal relatively easily using an automation and some extra research/trial and error with my limited knowledge.

After messing with it a bit, I ended up implementing it by using a helper to create a new input_number entity and only allowing 5-100 rather than 1-100 to eliminate that undesirable extra "1".    That new entity will then update the fan speed entity
